templates/zerver/keyboard_shortcuts.html

```diff
@@ -65,10 +65,14 @@
           <td class="definition">{% trans %}New private message{% endtrans %}</td>
         </tr>
         <tr>
-          <td class="hotkey">Tab then Enter</td>
+          <td class="hotkey"><p>Tab then Enter,</p><p>Ctrl then Enter</p></td>
```

Comment:
I think it's actually Ctrl-Enter (no then), since you need to not release the `Ctrl` key.  Same with Shift below.